app/schema.graphql

```diff
@@ -247,27 +370,39 @@ type Dimension implements Node {
     timeRange: TimeRange!
     granularity: Granularity!
 
-    """The inferences (primary or reference) to query"""
+    """
+    The inferences (primary or reference) to query
+    """
     inferencesRole: InferencesRole = primary
   ): DataQualityTimeSeries!
 
   """
   The time series of the specified metric for data within a time range. Data points are generated starting at the end time and are separated by the sampling interval. Each data point is labeled by the end instant and contains data from their respective evaluation windows.
   """
-  driftTimeSeries(metric: ScalarDriftMetric!, timeRange: TimeRange!, granularity: Granularity!): DriftTimeSeries!
+  driftTimeSeries(
+    metric: ScalarDriftMetric!
+    timeRange: TimeRange!
+    granularity: Granularity!
+  ): DriftTimeSeries!
 
   """
   The segments across both inference sets and returns the counts per segment
   """
   segmentsComparison(primaryTimeRange: TimeRange): Segments!
 }
 
-"""A connection to a list of items."""
+"""
+A connection to a list of items.
+"""
```

Comment:
This diff is strange. Is it due to a new version of strawberry?

Reply:
It might be prettier. Let me re-run